authorization-service/handler.js

```diff
@@ -7,14 +7,15 @@ module.exports.basicAuthorizer = async (event, _, callback) => {
     const authorizationToken = event.authorizationToken;
     const encodedCreds = authorizationToken.split(" ")[1];
     const buff = Buffer.from(encodedCreds, "base64");
-    const plainCreds = buff.toString("utf-8").split(":");
-    const username = plainCreds[0];
-    const password = plainCreds[1];
+    const [username, password] = buff.toString("utf-8").split(":");
 
-    const storedUserPassword = process.env[username];
+    const storedUserPassword = process.env.elenapetra;
```

Comment:
Is there a reason you changed `const storedUserPassword = process.env[username];`? It looked fine to me and didn't hardcode your name into the code, which you'll be getting in the username variable anyway. 

(I'm assuming you have `elenapetra=TEST_PASSWORD` in the .env file)